code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -40,42 +40,38 @@
 				to_chat(src, "<span class='danger'>[pick("The pain is excruciating", "Please, just end the pain", "Your whole body is going numb")]!</span>")
 			blur_eyes(2)
 			stuttering = max(stuttering, 5)
+			adjust_stagger(1, FALSE, 1)
+			add_slowdown(1)
 		if(60 to 79)
-			if(!lying_angle && prob(20))
-				emote("me", 1, " is having trouble standing.")
 			blur_eyes(2)
 			stuttering = max(stuttering, 5)
-			if(prob(2))
-				if(!lying_angle)
-					emote("me", 1, " is having trouble standing.")
+			adjust_stagger(3, FALSE, 3)
+			add_slowdown(3)
+			if(prob(20))
 				to_chat(src, "<span class='danger'>[pick("The pain is excruciating", "Please, just end the pain", "Your whole body is going numb")]!</span>")
```

Comment:
Why remove the emotes?

Reply:
May as well get rid of the spam, especially when the stun elements have been removed at this level.

Reply:
This isnt problematic rng and I see no reason for removing them, add them back